doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -892,27 +901,27 @@ Binding arguments of a constant to an interpretation scope
    turn to be themselves arguments of a reference are interpreted
    accordingly to the arguments scopes bound to this reference.
 
-.. cmdv:: Arguments @qualid : clear scopes
+   .. cmdv:: Arguments @qualid : clear scopes
 
-   Arguments scopes can be cleared with :n:`Arguments @qualid : clear scopes`.
+      Arguments scopes can be cleared with :n:`Arguments @qualid : clear scopes`.
```

Comment:
Maybe this sentence would be worth to reformulate, because like this it's pretty useless.